lua/net.lua

```diff
@@ -1,10 +1,14 @@
 local gatekeeperc = require("gatekeeperc")
+local ffi = require("ffi")
 
 local M = {}
 
 -- Function that sets up the network.
 function M.setup_block()
 
+	local front_ports = {"0000:83:00.0"}
+	local back_ports = {"0000:85:00.0"}
```

Comment:
The `setup.sh` should write it at the folder "lua" of the repository. And our Lua code can assume the existence of the file.